mmeds/resources/summary_code.txt

```diff
@@ -88,6 +88,54 @@ df = pd.read_csv('otu_table.tsv', skiprows=1, header=0, sep='\t')
 df.set_index('taxonomy', inplace=True)
 df.iloc[:5, :5]
 =====
+table_py<source>
+df = pd.read_csv('{file1}', sep='\t', header=0, skiprows=[1])
+data = []
+m = 0
+# Split rows into five categories
+for i in range(len(df['sample-id'])):
+    _input = df['input'][i]
+    _filtered = df['filtered'][i]
+    _denoised = df['denoised'][i]
+    _merged = df['merged'][i]
+    _nonchimeric = df['non-chimeric'][i]
+    # Store maximum
+    if _input > m:
+        m = _input
+    # Order sections so less filtered goes at the top
+    data.append([df['sample-id'][i], _input - _filtered, "a"])
+    data.append([df['sample-id'][i], _filtered - _denoised, "b"])
+    data.append([df['sample-id'][i], _denoised - _merged, "c"])
+    data.append([df['sample-id'][i], _merged - _nonchimeric, "d"])
+    data.append([df['sample-id'][i], _nonchimeric, "e"])
+df = pd.DataFrame(data, columns=['variable', 'value', "type"])
+# Create discrete y axis vals
+count = 5
```

Comment:
why 5?

Reply:
I’m manually generating the y-axis labels so they look a certain way. tried a couple of values for ‘count’, which is the number of y-axis number labels minus one, and 5 looked the best.